pages/visualizations/overview/pr_staleness.py

```diff
@@ -190,10 +185,13 @@ def new_staling_prs(repolist, timer_pings, interval, staling_interval, stale_int
     # create dataframe from record data
     df = pd.DataFrame(results)
 
-    # change all to datetime
```

Comment:
same questions as above. why try all of this instead of just testing if the results/ df is empty?

Reply:
same answer above